### PR TITLE
add java test case for oblogproxy-ce

### PR DESCRIPTION
**Pull request by whhe**

### CODE REVIEW

File: .github/workflows/test-oblogproxy-ce.yml [hunk 1]
```diff
@@ -0,0 +1,126 @@
+name: test oblogproxy-ce
+
+on:
+  push:
+    branches: [ main ]
```
Comment (chris-sun-star) [inline .github/workflows/test-oblogproxy-ce.yml:5]: is it necessary to test on main branch push

File: .github/workflows/test-oblogproxy-ce.yml [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  group: test-oblogproxy-ce-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build-oceanbase-ce:
+    uses: ./.github/workflows/build-oceanbase-ce.yml
+    with:
+      cache_key: test-oblogproxy-ce_oceanbase-ce
+      image_file: oceanbase-ce.tar
+      version: 4.2.1.6-106000012024042515
```
Comment (chris-sun-star) [inline .github/workflows/test-oblogproxy-ce.yml:24]: the latest version of oceanbase-ce 4.2.1 is 4.2.1.7-107000162024060611

Comment (whhe) [reply .github/workflows/test-oblogproxy-ce.yml:24]: The legacy version here is used for cdc libary testing.

**Review by chris-sun-star — APPROVED** [review]
LGTM

**Pull request merged**